start.go

```diff
@@ -44,7 +49,7 @@ func start(c *cli.Context) error {
 		"--name", "fnserver",
 		"-v", fmt.Sprintf("%s/data:/app/data", wd),
 		"-v", "/var/run/docker.sock:/var/run/docker.sock",
-		"-p", "8080:8080",
+		"-p", fmt.Sprintf("%d:8080", c.Int("port")),
```

Comment:
This is not enough, what if the user would use different service port? So, this port binding will make Fn server unreachable.

Reply:
With recently merged code, you need to check env file for FN_PORT var and set exact service port to build the port binding, if FN_PORT is not mentioned - use default one.